### PR TITLE
Adding params to http get for the proxy server

### PR DESCRIPTION
**Pull request by arsidada**

### CODE REVIEW

File: src/server.js [hunk 1]
```diff
@@ -62,11 +63,12 @@ if (process.env.NODE_ENV && process.env.NODE_ENV !== 'development') {
   });
 }
 
-function httpGet(method, path, headers) {
+function httpGet(method, path, headers, params) {
```
Comment (ahmedhamodi) [inline src/server.js:66]: Should we set a default value on params, like `params = null` so that the other calls to httpGet without params work?

**Review by ahmedhamodi — APPROVED** [review]
I left a comment that may need to be addressed, but otherwise LGTM.

File: src/server.js (hunk 1 above)
Comment (arsidada) [reply src/server.js:66]: all incoming requests are sent to the handler here: https://github.com/maxgala/aspire-frontend/pull/230/files#diff-79f4c7cdc0b026f6e556196f5911a449157807cffdb3f9d33f0b5404b4ffc2f3R18

that's where we get `req.query` and then pass into the `httpGet` helper. I just printed out what `req.query` looks like if no parameters are provided and it turns out to be an empty object `{}` which axios is able to handle the same way as a `null`

Comment (ahmedhamodi) [reply src/server.js:66]: Sounds good, thanks for checking that

**Pull request merged**